mmedit/datasets/sr_reds_dataset.py

```diff
@@ -83,7 +83,7 @@ def load_annotations(self):
                 dict(
                     lq_path=self.lq_folder,
                     gt_path=self.gt_folder,
-                    key=key,
+                    key=key.split('(')[0].strip(),
```

Comment:
I think the key will not contain '('. What is the case?

Reply:
The key is loaded from annotation file, which is `000_00000000.png (720,1280,3)` for example. 
But we need key=`000_00000000.png`.

The annotation file generate function:
https://github.com/wwhio/mmediting/blob/9a93413a10cf208d7a08d4db0e7937c6adae6a2d/tools/preprocess_reds_dataset.py#L137

Reply:
I check this again. A line in anno_file is `259/00000053 (720, 1280, 3)`, and key is `259/00000053 (720, 1280, 3)` before modification.

Reply:
OK, I think a better way is to modify: 

https://github.com/open-mmlab/mmediting/blob/9a93413a10cf208d7a08d4db0e7937c6adae6a2d/mmedit/datasets/sr_reds_dataset.py#L61-L64

maybe `keys = [v.strip().split('.')[0] for v in fin]` should be `keys = [v.strip().split(' ')[0] for v in fin]`

Reply:
BTW, I think a better way of organizing video annotation file is like this:

https://github.com/xinntao/BasicSR/blob/f33864a7538496a6ded8aa1d75d68df6a91f5193/basicsr/data/reds_dataset.py#L64-L68

https://github.com/xinntao/BasicSR/blob/f33864a7538496a6ded8aa1d75d68df6a91f5193/basicsr/data/meta_info/meta_info_REDS_GT.txt#L1-L26


It will keep the annotation file small and clean.

Reply:
I follow your first suggestion to modify `mmediting/mmedit/dataset/sr_reds_dataset.py` to keep the modification small and keep in consistent with vimeo90k dataset.

Reply:
I roll back changes to `mmediting/mmedit/dataset/sr_reds_dataset.py` and some related files. The real problem lies in the generated annotation file.